public/index.html

```diff
@@ -2,7 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Excalidraw</title>
+    <title>
+      Excalidraw | Virtual collaborative whiteboard for sketching hand-drawn
+      like diagrams
```

Comment:
@vjeux if you have a better suggestion for the tagline... and what separator do you guys prefer?